widgetssdk/src/main/java/com/glia/widgets/entrywidget/adapter/EntryWidgetAdapter.kt

```diff
@@ -6,11 +6,14 @@ import androidx.recyclerview.widget.RecyclerView
 import com.glia.widgets.databinding.EntryWidgetErrorItemBinding
 import com.glia.widgets.databinding.EntryWidgetMediaTypeItemBinding
 import com.glia.widgets.databinding.EntryWidgetPoweredByItemBinding
+import com.glia.widgets.di.Dependencies
 import com.glia.widgets.entrywidget.EntryWidgetContract
 import com.glia.widgets.helper.layoutInflater
+import com.glia.widgets.view.unifiedui.theme.entrywidget.EntryWidgetTheme
 
 internal class EntryWidgetAdapter(
-    private val viewType: EntryWidgetContract.ViewType
+    private val viewType: EntryWidgetContract.ViewType,
+    private val entryWidgetTheme: EntryWidgetTheme?
```

Comment:
This won't work in SC.
What do you think about passing properties one by one, in this case, we can pass `null` for error states in SC because we will hide the `live support` button if there is no available media type.

Reply:
fixed